Dockerfile

```diff
@@ -45,8 +47,7 @@ ENV HOME=/viya4-deployment
 RUN pip install -r ./requirements.txt \
   && ansible-galaxy install -r ./requirements.yaml \
   && chmod -R g=u /etc/passwd /etc/group /viya4-deployment/ \
-  && chmod 755 /viya4-deployment/docker-entrypoint.sh \
-  && git config --system --add safe.directory /viya4-deployment
```

Comment:
Why was this removed? If someone clones this repo it'll have a value.

Reply:
I removed line 49 since for this Dockerfile the /viya4-deployment folder is not a git managed folder and when I ran "docker build" with that line in place, I got a git error indicating that. Removing the line allowed me to build the container. Could it be an issue with the version of git I'm using?

Reply:
@thpang, I restored the deleted line after our discusion, ready for re-review